stacks/Chart.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: stacks
+description: A developer-first application format for running containers in Kubernetes
```

`A developer-first application format` doesn't say anything to me on it's own. I'd go with something more direct. 

```suggestion
description: Docker-compose for Kuberntes
```